src/main/java/dota/buff/model/User.java

```diff
@@ -11,5 +11,5 @@
 public class User {
     String login;
     String password;
-    Key key;
+    long steamId;
```

Comment:
I'm thinking ab - will the user have an opportunity to change steamId, shouldn't it be a final?
Relates to login and password, I guess it could be changed by the user optionally.

Reply:
No, because account is not belong to steamId, the user can change steam and continue with our service account